pulp_rpm/app/kickstart/treeinfo.py

```diff
@@ -134,7 +134,7 @@ def parsed_sections(self):
         tree = self.original_parser._sections.get("tree", {})
 
         if "." in str(tree.get("build_timestamp")):
-            build_timestamp = float(tree.get["build_timestamp"])
+            build_timestamp = float(tree.get("build_timestamp"))
```

Comment:
Oh my... :woman_facepalming: 
We need to add a repo with such timestamp to our testing suite. @jlsherrill , do you have an idea which repos the user has the problem with?

Reply:
The user here that filed this issue: https://pulp.plan.io/issues/5989  said centos7 has this issue, but i couldn't seem to find a centos7 repo that had it: https://mirror.linux.duke.edu/pub/centos/7/os/x86_64/.treeinfo

Reply:
CentOS 8 has it, e.g. http://mirror.centos.org/centos/8/BaseOS/x86_64/kickstart/.treeinfo

Reply:
> CentOS 8 has it, e.g. http://mirror.centos.org/centos/8/BaseOS/x86_64/kickstart/.treeinfo

@gvde , thanks, the build_timestamp we can find but I'm looking for a timestamp which will be of a floating point type, not the integer. 
E.g. build_timestamp = 1605735523.0

Reply:
@goosemania Sorry. I have missed that detail. For what it's worth, looking at the latest mirrored repositories: CentOS 7 doesn't have any build_timestamp. CentOS 8.3.2011 and Stream have only integer. So no help there...